### DISCUSSION TITLE
Welcome / Changelog popup on first open

### DISCUSSION THREAD

**Discussion by gkeep** [wstyres/Zebra#762] [Ideas]
It would be nice to have a popup with changelog similar to Apollo for Reddit or Discord (see below) *especially* with the upcoming redesign.
<img src="https://external-preview.redd.it/mwJUz1SHL1JncxqzdxVLzP0HWsEyLawqVZ2SMtOhl1Y.jpg?auto=webp&s=371665d073965078fc7eb12a85fdf5f771e1b22f" width="300" height="530"></img><img src="https://user-images.githubusercontent.com/20600053/71642159-300d3d00-2cb8-11ea-8edf-9b0311ec6c39.jpeg" width="300" height="530"></img>

**Comment by worldlydev** [comment]
Cool idea. I personally think it should look more like Apple's own iOS apps after an iOS update.

Like with a "What's New" page.
https://developer.apple.com/design/human-interface-guidelines/ios/app-architecture/onboarding/

Like this:
![3D650B6B-B6C5-4155-AC91-9FCC08D5E17E](https://user-images.githubusercontent.com/51668635/71645351-4d053880-2cd7-11ea-8aea-3a82b00b4f20.jpeg)


No need for new icons for every release. Just use the same ones. Perhaps the Github logo, Zebra icon, changelog icon, etc.

**Comment by wstyres** [comment]
Not a bad idea, I think they would probably only show up for major releases 1.1, 1.2, etc. because I feel they would get annoying for beta releases.

**Comment by gkeep** [comment]
I mean, you would be pulling changelog from [releases at github](https://github.com/wstyres/Zebra/releases) (same as Changelog button on Home page), so it assumes it is a major release.

edit: Nevermind, I'm dumb. I forgot that all releases are there.

**Comment by worldlydev** [comment]
Just wanted to let you know about this since I just stumbled upon it.

![6846580D-3C06-40A4-BD94-821651F0A002](https://user-images.githubusercontent.com/51668635/72384026-f78c4b00-371c-11ea-81e1-96e34b77373a.png)

This is the popup that shows up on new versions of some apps after updating to the latest testflight beta.

**Comment by worldlydev** [comment]
From Installer 🤫
![914589DD-9D69-4B9A-AB35-1671E236335B](https://user-images.githubusercontent.com/51668635/72624005-68627b80-3946-11ea-9e20-85ecd73deed6.jpeg)

**Comment by wstyres** [comment]
We're going to push this back to 1.2 since the design we have in mind doesn't much fit with Zebra's current homepage design.
